docs/baremetalhost-states.md

```diff
@@ -20,8 +20,8 @@ for registration.
 
 An Externally Provisioned host was deployed using another tool and
 then a host object was created with a link to an existing Machine
```

Comment:
Not part of this change, but we don't require a link to a Machine object any more either.

Reply:
Fixed.